src/particleContainer/AutoPasContainer.cpp

```diff
@@ -159,7 +174,9 @@ bool AutoPasContainer::rebuild(double *bBoxMin, double *bBoxMax) {
 	_autopasContainer.setEvidenceFirstPrediction(_evidenceForPrediction);
 	_autopasContainer.setExtrapolationMethodOption(_extrapolationMethod);
 	_autopasContainer.init();
-	autopas::Logger::get()->set_level(autopas::Logger::LogLevel::debug);
+	_autopasContainerIsInitialized = true;
+	// if you want more AutoPas output enable this
+	// autopas::Logger::get()->set_level(autopas::Logger::LogLevel::debug);
```

Comment:
please make this settable using XML input!

Reply:
you mean the autopas log level?

Reply:
yes

Reply:
done 2d25d2c